homeassistant/components/zwave_js/config_flow.py

```diff
@@ -41,6 +47,12 @@
 STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required(CONF_URL, default=DEFAULT_URL): str})
```

Comment:
Can this be removed since we now generate a schema in `get_manual_schema`?

Reply:
Yes, I think I just forgot to remove it in this branch. 👍